InfinityCosmos/ForMathlib/AlgebraicTopology/SimplicialSet/MorphismProperty.lean

```diff
@@ -26,4 +27,35 @@ def trivialFibration : MorphismProperty SSet := fun _ _ p ↦ BoundaryInclusions
 
 end trivialFibration
 
+section isoFibration
+
+/-- Inductive definition of inner horn inclusions Λ[n, i] ⟶ Δ[n] 
+  by restricting general horn inclusions to 0 < i < n -/
+inductive InnerInclusion : {X Y : SSet} → (X ⟶ Y) → Prop
+  | mk (n i : ℕ) (low : 0 < i) (high : i < n) : InnerInclusion (hornInclusion n i)
+
+/-- The class of inner horn inclusions as a MorphismProperty -/
+def InnerInclusions : MorphismProperty SSet := fun _ _ p ↦ InnerInclusion p
+
+/-- Inductive definition of being equal to the coherent iso -/
+inductive CoherentIsoInclusion : {X Y : SSet} → (X ⟶ Y) → Prop
```

Comment:
We'll want to be able to lift against the inclusion of either 0 or 1. Perhaps it's best to have both as elements of this class? 

Then (later) you could prove that having the rlp against `coherentIso.pt .zero` implies the same for `coherentIso.pt .one` by a retract argument. Then you could have a lemma that helps you define an isofibration from either lifting property.

Reply:
> We'll want to be able to lift against the inclusion of either 0 or 1. Perhaps it's best to have both as elements of this class?
> 
> Then (later) you could prove that having the rlp against `coherentIso.pt .zero` implies the same for `coherentIso.pt .one` by a retract argument. Then you could have a lemma that helps you define an isofibration from either lifting property.

Sure, I guess my thinking was to minimize the requirements to define it to make constructing it easier, but then of course using it is more difficult.